source/module_esolver/esolver_sdft_pw.cpp

```diff
@@ -171,6 +171,7 @@ void ESolver_SDFT_PW<T, Device>::hamilt2density_single(UnitCell& ucell, int iste
                               this->psi[0],
                               this->pelec,
                               this->pw_wfc,
+                              ucell,
```

Comment:
put this as the first parameter

Reply:
This parameter has already been set as the first parameter.